### DISCUSSION TITLE
[Performance] : Maximum limit Of agents a single Collector can hold

### DISCUSSION THREAD

**Discussion by the-rough-page** [apache/skywalking#8766] [Q&A] [answered]
If we are opting for java, is there any limitations for the Single Collector to Handling numbers of agents?
How much at max we can handle?

**Comment by wu-sheng** [comment, marked as answer]
Officially, we don't have this number. This depends on the number of spans per trace, the traffic, the storage you chose.

**Reply by the-rough-page** [reply]
any Maximum Tried Number yet ?

**Reply by wu-sheng** [reply]
At least I don't have that.
